pkgs/by-name/ch/choparp/package.nix

```
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  libpcap,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "choparp";
+  version = "0-unstable-2021-04-23";
+
+  src = fetchFromGitHub {
+    owner = "quinot";
+    repo = "choparp";
+    rev = "e9f0b81135d81cb0416504a7e695e158f4a5285e";
+    hash = "sha256-0VZj7Hkn/aiRddWdBzDAXdOdxJZvwd+KaN9ddrrBjm8=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    mkdir -p $out/bin
+    $CC -o $out/bin/choparp src/choparp.c -lpcap -L${lib.getLib libpcap}/lib -I${libpcap}/include
```

```suggestion
    $CC -o $out/bin/choparp src/choparp.c -lpcap -L${lib.getLib libpcap}/lib -I${lib.getDev libpcap}/include
```
Or is this the default?